mods/ra/mod.yaml

```diff
@@ -8,15 +8,16 @@ RequiresMods:
 	modchooser: {DEV_VERSION}
 
 Packages:
-	~^Content/ra
-	~^Content/ra/music
+	~^Content/ra/v2/
+	~^Content/ra/v2/expand
+	~^Content/ra/v2/cnc
 	.
 	$ra: ra
 	$cnc: cnc
 	./mods/common: common
 	~main.mix
-	~redalert.mix
```

Comment:
one of the pr goals is to guarantee a standard set of installed assets.  Calling this a feature might be a bit far, but it's certainly not an unwanted side effect.